zScanner/Views/NewDocument/PhotoSelectorCollectionViewCell.swift

```diff
@@ -56,14 +56,16 @@ class PhotoSelectorCollectionViewCell: UICollectionViewCell {
     private func setupView() {
         contentView.addSubview(imageView)
         imageView.snp.makeConstraints { make in
-            make.edges.equalToSuperview()
+            make.top.trailing.leading.equalToSuperview()
+            make.height.equalTo(imageView.snp.width)
         }
         
         textField.delegate = self
         contentView.addSubview(textField)
         textField.snp.makeConstraints { make in
-            make.height.equalTo(35)
-            make.width.equalToSuperview()
+            make.top.equalTo(imageView.snp.bottom).offset(4)
+            make.height.equalTo(36)
+            make.leading.trailing.equalToSuperview().inset(4)
```

Comment:
The text field was covering the image, instead of being under the image. I have fixed the constraints and added padding.